catalog-item.html

```diff
@@ -232,12 +248,12 @@ <h3 class="main-content-heading">SUBSCRIBE</h3>
                     <hr>
                     <p class="copyright-text">BINABOX 2022 - All rights reserved</p>
                     <ul class="social-links">
-                        <li><a href="#" class="social-link">Facebook</a></li>
-                        <li><a href="#" class="social-link">Twitter</a></li>
-                        <li><a href="#" class="social-link">Telegram</a></li>
-                        <li><a href="#" class="social-link">YouTube</a></li>
-                        <li><a href="#" class="social-link">TikTok</a></li>
-                        <li><a href="#" class="social-link">Discord</a></li>
+                        <li><a href="#" target="_blank" class="social-link"><img src="Images\icons\twitter.svg" alt="twitter icon" width="20px" height="20px"><i class="twitterlink"></i></a></li>

```

Comment:
You're absolutely right! The <i> tag is semantically intended for italicizing text, so it's not the best choice for icons. Instead, it's better to use a <span> tag or, even better, an <svg> or <img> tag specifically for icons.